### ISSUE TITLE
Question about training

### ISSUE THREAD

**Issue by Joel18241096** [lzzcd001/MeshDiffusion#17]
Hi, i have downloaded your preprocessed dataset and wish to execute the training process, but i have no idea on how to generate the meta json file, could you give me some hint please?

**Comment by lzzcd001** [comment]
The JSON file is basically a list of all training file paths in the training set in JSON format. Please check `save_meta.py`.

**Comment by Joel18241096** [comment]
I have downloaded your preprocessed dataset, which is npy, but your save_meta.py and dataloader requires pt as input. Is the cubic data in your npy file can be directly used in training? thanks!

**Comment by lzzcd001** [comment]
The dataset you downloaded is processed tets in tetrahedral grids, not cubic ones. You have to run `tets_to_3dgrid.py` to turn stored tets into cubic grids.

**Comment by Joel18241096** [comment]
The dataset i have downloaded is a npy file, where the numpy array is in shape of (4, 64, 64, 64). Your tets_to_3dgrid.py file requires to take in a pt file as input, which contains two keys: 'sdf' and 'deform'. I assume that i need to process each npy file, where the first value of axis 0 represents sdf, and the axis 1 to 3 represents deform of each grids?

**Comment by lzzcd001** [comment]
Yes, but I was wrong about the dataset - they are already processed cubic grids in `.npy` files. Check the latest commits and README for details, or you can write your own dataloader if you find it simpler.

**Comment by Joel18241096** [comment]
thanks!
